### PR TITLE
Add flag size, and use root navigator

### PR DESCRIPTION
**Pull request by mohn93**
Add flag size, and use root navigator

### CODE REVIEW

File: lib/src/widgets/country_selector/country_list.dart [hunk 1]
```diff
@@ -38,7 +39,8 @@ class CountryList extends StatelessWidget {
     required this.noResultMessage,
     this.scrollController,
     this.scrollPhysics,
-    this.showDialCode = true,
+    this.showDialCode = false,
```
Comment (cedvdb) [inline lib/src/widgets/country_selector/country_list.dart:42]: why did you change the default ?

Comment (mohn93) [reply lib/src/widgets/country_selector/country_list.dart:42]: Oh Sorry, I was play around with params and forgot to reset this

Comment (mohn93) [reply lib/src/widgets/country_selector/country_list.dart]: Reopening new pull request

Comment (mohn93) [reply lib/src/widgets/country_selector/country_list.dart]: Fixed by adding new commit

**Comment by mohn93** [comment]
Is there anything you want me to do, so you can merge?

**Comment by cedvdb** [comment]
published
